spec/models/bike_spec.rb

```diff
@@ -1335,7 +1335,7 @@
       end
       let(:bike) { FactoryBot.create(:stolen_bike) }
       it "does not get out of integer errors" do
-        expect(bike.listing_order).to be < 10000
+        expect(bike.listing_order).to be < 1e10
```

Comment:
While this is the limit for out of range, I feel like testing that it's well below the limit is good.

If you'd prefer `1e4` here, that's fine.

Reply:
No prob  — reverted in dea790b1
The test currently fails intermittently but it's pretty rare for it to exceed `10_000`.

Reply:
If it is actually flaky, maybe switch it to 1e5? I would much prefer non-flaky, I just also think having some extra space up there would be great.